pyproject.toml

```
@@ -84,6 +83,7 @@ format-check = "ruff format --check {args}"
 
 [tool.hatch.envs.test]
 extra-dependencies = [
+  "pandas",                                    # Used in XLSXToDocument, CSVDocumentSplitter,  CSVDocumentCleaner, EvaluationRunResult and pipeline tests
```

Forgot AzureOCRDocumentConverter earlier, I'm sorry.
```suggestion
  "pandas",                                    # Used in XLSXToDocument, CSVDocumentSplitter,  CSVDocumentCleaner, AzureOCRDocumentConverter, EvaluationRunResult and pipeline tests
```